client/components/popover/util.js

```diff
@@ -38,42 +38,45 @@ const adjacent = {
 	right: 'bottom',
 };
 
-//let viewport;
+let _viewport = null;
```

Comment:
The truth is that I first renamed the variable to `_viewport` and then ESLint showed me all the usages of the original `viewport` variable in the file. And I replaced these with `getViewport()`.

I can change that back, since the `_` prefix did it's job already 🙂

Reply:
The `_viewport` name has one more benefit: it doesn't conflict with local variables named `viewport` in functions that call `getViewport()`. So, to avoid too many changes, I'll keep the names prefixed.